utils_test.go

```diff
@@ -116,3 +116,35 @@ func TestContains(t *testing.T) {
 		}
 	})
 }
+
+func TestGetConnectionId(t *testing.T) {
+	for i := 0; i < 100; i++ {
```

Comment:
why do we run it 100 times? is there something specific about this SDK compared to the others where we test in only once?